README.md

```diff
@@ -419,6 +419,60 @@ RESULTS
 |---------|------------------------|--------------------------------|--------------------------------|---------------------------------------------------------------------------|
 ```
 
+##### Showing Maintainers Annotation
+
+To see maintainers annotations for each check, use the `--show-ma` option. The annotations are configured as labels but parsed to a human-readable format in the output.
```

Comment:
Style question:

We talk about maintainer annotations, but in the code should we just refer to it as `annotations`?

e.g. `package annotations` or cli flag: `--show-annotations`